front/app/utils/dateUtils.test.ts

```diff
@@ -0,0 +1,187 @@
+import {
```

Comment:
Yeah, since we're not passing in a valid timezone, moment defaults to what it assumes is correct. To check this you can do `moment.tz.guess()` (likely in index.tsx, before anything has been set) and check the return value, which should be your local timezone or thereabouts.